### PR TITLE
[ACR-1318] add new features

### PR DESCRIPTION
**Pull request by jinwangan**
ACR-1318] add new features

     - http://tizen.org/feature/accessibility.grayscale
     - http://tizen.org/feature/accessibility.negative

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/591

File: docs/application/native/guides/device/system.md [hunk 1]
```diff
@@ -155,6 +156,17 @@ The following table lists the account feature keys:
...
 | `http://tizen.org/feature/account.sync` | `bool` | The platform returns `true` for this key, if the device supports the Sync Manager API. |
 
+The following table lists the accessibility feature keys:
+
+<a name="accessibility"></a>
+**Table: Accessibility feature keys**
+
+| Key                                     | Type   | Description                              |
+|-----------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/accessibility.grayscale`` | `bool` | The platform returns `true` for this key, if the device supports the accessibility grayscale feature. |
```
Comment (journey2w) [inline docs/application/native/guides/device/system.md]: Remote one  tick mark (`) at the end.
```
`http://tizen.org/feature/accessibility.grayscale`
```

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/591

File: docs/application/native/guides/device/system.md (hunk 1 above)
Comment (jinwangan) [reply docs/application/native/guides/device/system.md]: done

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/591

**Pull request merged**